frontend/src/components/Appbar.tsx

```diff
@@ -18,7 +18,7 @@ const Appbar = () => {
           <Link to="/publish">
             <button
               type="button"
-              className="focus:outline-none text-white bg-green-700 hover:bg-green-800 focus:ring-4 focus:ring-green-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-green-600 dark:hover:bg-green-700 dark:focus:ring-green-800 mx-12"
+              className="focus:outline-none text-white bg-green-700 hover:bg-green-800 focus:ring-4 focus:ring-green-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2"
```

Comment:
Was `mx-12` removed intentionally ? If no, then please add it again.

Reply:
Hey @aadeshkulkarni, 
added mx-12, please check it and confirm.